prow/monitoring/grafana_configmaps.yaml

```
@@ -1,6 +1,36 @@
 ---
 apiVersion: v1
 kind: ConfigMap
+metadata:
+  namespace: prow-monitoring
+  name: grafana-nginx
+data:
+  nginx.conf: |
+    pid        /tmp/nginx.pid;
+
+    events { }
+
+    http {
+        client_body_temp_path /tmp/client_temp;
+        proxy_temp_path       /tmp/proxy_temp_path;
+        fastcgi_temp_path     /tmp/fastcgi_temp;
+        uwsgi_temp_path       /tmp/uwsgi_temp;
+        scgi_temp_path        /tmp/scgi_temp;
+
+        server {
+            listen 80 default_server;
+            server_name _;
+            if ($http_x_forwarded_proto != 'https') {
+                return 301 https://monitoring.prow.k8s.io;
```

This can be generic:
```suggestion
                return 301 https://$host$request_uri;
```

---

Sorry for the late reply, @cblecker 
Just came back from PTO.